src/java.desktop/share/classes/javax/swing/plaf/synth/SynthSplitPaneUI.java

```diff
@@ -347,12 +363,21 @@ private void paintDragDivider(Graphics g, int x, int y, int w, int h) {
     public void finishedPaintingChildren(JSplitPane jc, Graphics g) {
         if(jc == splitPane && getLastDragLocation() != -1 &&
                               !isContinuousLayout() && !draggingHW) {
+            Dimension      size = splitPane.getSize();
+
+            g.setColor(dividerDraggingColor);
             if(jc.getOrientation() == JSplitPane.HORIZONTAL_SPLIT) {
```

Comment:
I know these if's already existed, but now might be a good time to also update these by adding a space after the if's to match the space after the else?